sites/website/src/docs/fast-foundation/getting-started.md

```diff
@@ -2,4 +2,87 @@
 title: Getting Started
 sidebar_label: Getting Started
 custom_edit_url: https://github.com/microsoft/fast-dna/edit/master/sites/website/src/docs/fast-foundation/getting-started.md
----
\ No newline at end of file
+---
+This document presumes you have [NodeJS](https://nodejs.org/) installed as well as some way to run an HTTP server locally, such as [http-server](https://www.npmjs.com/package/http-server).
+
+## Quick-start
+This section will walk through the quickest way to get up and running using FAST components.
+
+### Add the script
+Next, add the `fast-components.js` script file to your HTML. This pre-bundled script contains all components and dependencies. Also be sure to add `type="module"` to the script element.
```

Comment:
Could we elaborate on the benefits of `type=module`? Here's a link that may be worth linking too. https://stackoverflow.com/questions/51418964/script-type-text-javascript-vs-script-type-module

Reply:
yea ill add a link to what that type does